### ISSUE TITLE
3.3 Input Assistance

### ISSUE THREAD

**Issue by DonnaDuiker** [alkem-io/client-web#8376]
# Background
After going through the WCAG 2.1 checklist, creating issues for all A and AA points that still need work. 1 issue for each guideline.

This is guideline 3.3 Input Assistance

# Acceptance Criteria

- [ ] 3.3.1 Error Identification: Errors are clearly identified and described. (More work needed: f.e. Within input text is too long, not everywhere it is identified and described + when adding an event and times mismatch, you cannot add but don't know why.)
- [ ] 3.3.3 Error Suggestion: Suggestions are provided for input errors.	(We don't do suggestions (f.e. When text is too long, descrease text size))

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)
